src/CodeStyle/CSharp/Analyzers/EditorConfigOptionsApplier.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.VisualStudio.CodingConventions;
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal class EditorConfigOptionsApplier
+    {
+        public OptionSet ApplyConventions(OptionSet optionSet, ICodingConventionsSnapshot codingConventions, string languageName)
```

Comment:
languageName doesn't appear to be used.

Reply:
Not used in the code style layer, but many places in the shared code pass it as an argument.

Reply:
Oh nevermind, yes, seems like this can be simplified now.